### ISSUE TITLE
Automatically add dead token's on combat's scene to tracker for drops

### ISSUE THREAD

**Issue by finderski** [trioderegion/nova#9]
At the end of a Round, if you click the Advance Round button, all the GM controlled tokens disappear from the Tracker, and they need to be added back again.

**Comment by trioderegion** [comment]
This is expected behavior. I'm sure the UX can be improved, but the idea is that the NPCs that the GM moves each turn will be different, since the GM is limited to moving `num PCs` of NPCs per turn.

Technically speaking, the GM doesn't actually need to add NPCs that have acted to the combat tracker if that is easier. It is provided as a reminder.

I hope that makes sense 😅.

**Comment by finderski** [comment]
Yeah, it makes sense, I found I just added all the enemies, because it was easier to do the drops…especially since my players would target enemies that weren’t acting, etc. Rich Finder"Nice tuxedo! Nice tuxedo to die in!"On Jun 3, 2023, at 10:23 AM, trioderegion ***@***.***> wrote:﻿
This is expected behavior. I'm sure the UX can be improved, but the idea is that the NPCs that the GM moves each turn will be different, since the GM is limited to moving num PCs of NPCs per turn.
Technically speaking, the GM doesn't actually need to add NPCs that have acted to the combat tracker if that is easier. It is provided as a reminder.
I hope that makes sense 😅.
